packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts

```diff
@@ -187,7 +187,7 @@ export const getStyles = (props: IPivotStyleProps): IPivotStyles => {
             right: 0,
             bottom: 0,
             content: '""',
-            height: 'auto',
+            height: 0,
```

Comment:
Also update Pivot.scss to keep the sass styles in sync with the js styles.

@xugao is there a plan to keep the two copies of these styles in sync in general?

Reply:
as of now, i don't think we need the scss for the long term (we are going with `merge-styles`). however we kept them around in case we want to compare perf. To answer your question directly, no we don't need to keep them exactly in sync.